optional/e2e/.runners/puppeteer/step_definitions/global.js

```diff
@@ -35,6 +39,14 @@ Quand("j'attends {string} secondes", duration => {
   I.wait(parseInt(duration, 10));
 });
 
+Quand("j'attends que les résultats de recherche apparaissent", () => {
+  I.waitForElement("[aria-label^='Résultats de recherche']", 5);
+});
+
+Quand("j'attends que le titre de page {string} apparaisse", title => {
+  I.waitForElement(`//h1[text()="${title}"]`)
```

Comment:
using xpath instead of css selector ?

Reply:
The only spec allowing to select a element according to its text content has been deprecated according to W3C:/
https://stackoverflow.com/questions/45955239/css3-contains-selector-not-work
https://stackoverflow.com/questions/1520429/is-there-a-css-selector-for-elements-containing-certain-text